templates/contribute/style.md

```diff
@@ -392,6 +392,21 @@ We generally use a blank line to separate theorems and definitions,
 but this can be omitted, for example, to group together a number of
 short definitions, or to group together a definition and notation.
 
+### Hypotheses Left of Colon ###
```

Comment:
I think this should go earlier in the document, maybe right before the `### Binders ###` section.

Otherwise LGTM.